src/lib/consistentAssetSize.ts

```diff
@@ -0,0 +1,71 @@
+import { mapValues, range, zipObject } from 'lodash';
+
+import { SUPPORTED_LOCALE_STRING_LABELS, SupportedLocales } from '@/constants/localization';
+
+import { formatNumberOutput, OutputType } from '@/components/Output';
+
+// for each locale, an array of the correct compact number suffix to use for 10^{index}
+// e.g. for "en" we have ['', '', '', 'k', 'k', 'k', 'm', 'm', 'm', 'b', 'b', 'b', 't', 't', 't']
```

Comment:
Does anything break if, say, we need to represent a number over 1,000T? Or does it handle this gracefully and display 10,000T?

Reply:
Yep that's what it does.